shot-android/src/test/java/com/karumi/shot/compose/ComposeScreenshotTest.kt

```diff
@@ -26,7 +27,7 @@ class ComposeScreenshotTest {
 
     @Before
     fun setUp() {
-        composeScreenshot = ComposeScreenshot(ScreenshotTestSession.empty, screenshotSaver)
+        composeScreenshot = ComposeScreenshot(ScreenshotTestSession(), screenshotSaver)
```

Comment:
This was an interesting discovery: `empty` is a single instance, so it's not actually empty anymore after it's mutated. All of the tests comparing equality were simply comparing equality of the exact same instance.

So I've deprecated `empty` since it does not behave as expected, and I've updated to the tests to compare the members of the session (which are data classes) rather than the session itself (which does not implement `equals`). Let me know if you'd like it handled differently.

Reply:
You are absolutely right. I didn't notice it. Thanks!